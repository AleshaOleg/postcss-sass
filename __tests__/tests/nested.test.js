var getPostCssTreeFromSass = require('../../helpers/getPostCssTreeFromSassTree');

it('nested.sass', function () {
    expect(getPostCssTreeFromSass('nested')).toMatchSnapshot();
});
