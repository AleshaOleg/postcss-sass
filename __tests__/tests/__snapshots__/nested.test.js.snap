// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nested.sass 1`] = `
Object {
  "nodes": Array [
    Object {
      "nodes": Array [
        Object {
          "nodes": Array [
            Object {
              "prop": "color",
              "raws": Object {
                "before": "
        ",
                "between": ": ",
                "semicolon": false,
              },
              "source": Object {
                "end": Object {
                  "column": 18,
                  "line": 3,
                },
                "input": Input {
                  "css": ".nested
    background:
        color: red
        repeat: no-repeat
",
                  "id": "<input css 1>",
                },
                "start": Object {
                  "column": 9,
                  "line": 3,
                },
              },
              "type": "decl",
              "value": "red",
            },
            Object {
              "prop": "repeat",
              "raws": Object {
                "before": "
        ",
                "between": ": ",
                "semicolon": false,
              },
              "source": Object {
                "end": Object {
                  "column": 25,
                  "line": 4,
                },
                "input": Input {
                  "css": ".nested
    background:
        color: red
        repeat: no-repeat
",
                  "id": "<input css 1>",
                },
                "start": Object {
                  "column": 9,
                  "line": 4,
                },
              },
              "type": "decl",
              "value": "no-repeat",
            },
          ],
          "raws": Object {
            "before": "
    ",
            "between": "",
          },
          "selector": "background:",
          "source": Object {
            "end": Object {
              "column": 26,
              "line": 4,
            },
            "input": Input {
              "css": ".nested
    background:
        color: red
        repeat: no-repeat
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 1,
              "line": 2,
            },
          },
          "type": "rule",
        },
      ],
      "raws": Object {
        "before": "",
        "between": "",
      },
      "selector": ".nested",
      "source": Object {
        "end": Object {
          "column": 26,
          "line": 4,
        },
        "input": Input {
          "css": ".nested
    background:
        color: red
        repeat: no-repeat
",
          "id": "<input css 1>",
        },
        "start": Object {
          "column": 1,
          "line": 1,
        },
      },
      "type": "rule",
    },
  ],
  "raws": Object {
    "before": "",
    "semicolon": undefined,
  },
  "source": Object {
    "end": Object {
      "column": 26,
      "line": 4,
    },
    "input": Input {
      "css": ".nested
    background:
        color: red
        repeat: no-repeat
",
      "id": "<input css 1>",
    },
    "start": Object {
      "column": 1,
      "line": 1,
    },
  },
  "type": "root",
}
`;
